### ISSUE TITLE
golib/uptime: cannot find package

### ISSUE THREAD

**Issue by miwarin** [mackerelio/mackerel-agent-plugins#481]
I download v0.41.1.tar.gz. then make. I have error:

```
package github.com/mackerelio/golib/uptime: cannot find package "github.com/mackerelio/golib/uptime" in any of:
        /usr/pkg/go/src/github.com/mackerelio/golib/uptime (from $GOROOT)
        /home/rin/work/go/src/github.com/mackerelio/golib/uptime (from $GOPATH)
*** Error code 1

Stop.
make: stopped in /home/rin/work/mackerel-agent-plugins-0.41.1
```

error in:

```
% grep -r github.com/mackerelio/golib/uptime *
mackerel-plugin-uptime/lib/uptime.go:   "github.com/mackerelio/golib/uptime"
```

cause:

https://github.com/mackerelio/mackerel-agent-plugins/commit/44675298711d18241ddf12bf6cd5e84136f3c2f9#diff-d8b387232381bdddabe6d5abfe095dc9

change from 'golib/uptime'  to 'go-osstat/uptime'.

(I git clone current. then make. I dont have this error.)

solution:
please release v0.41.2

**Comment by itchyny** [comment]
Sorry for inconvenience, we'll release a new version in a few days.

**Comment by astj** [comment]
@miwarin We've released v0.42.0. I believe you don't encounter this error in the version.
